creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -3,17 +3,32 @@ const config = require('../../config')
 const utils = require('../../utils.js')
 const { MONITORS } = require('../../monitors/monitors')
 
+const MIN_NUBMER_OF_CPUS = 8 // 8 cpu
+const MIN_TOTAL_MEMORY = 15500000000 // 15.5 GB of RAM
+const MIN_FILESYSTEM_SIZE = 1950000000000 // 1950 GB of file system storage
+
 /**
  * Perform a basic health check, returning the
  * currently selected discovery provider (if any),
  * the current git SHA, and service version info.
  * @param {*} ServiceRegistry
  * @param {*} logger
  * @param {*} sequelize
+ * @param {*} getMonitors
+ * @param {number} numberOfCPUs the number of CPUs on this machine
  * @param {string?} randomBytesToSign optional bytes string to be included in response object
  *    and used in signature generation
  */
-const healthCheck = async ({ libs } = {}, logger, sequelize, randomBytesToSign = null) => {
+const healthCheck = async ({ libs } = {}, logger, sequelize, getMonitors, numberOfCPUs, randomBytesToSign = null) => {
+  // System information
+  const [
+    totalMemory,
+    storagePathSize
+  ] = await getMonitors([
+    MONITORS.TOTAL_MEMORY,
```

Comment:
i agree, there's nothing that should really affect health checks latency siginificantly since all the other verbose fields are pulled from memory or redis